Loadable.svelte

```diff
@@ -66,6 +68,7 @@
   let state = STATES.INITIALIZED
   let componentProps
   let slots = $$props.$$slots
+  let mounted = true;
```

Comment:
It starts as `false`

```suggestion
  let mounted = false;
```

Reply:
good call - fixed